# Currently available actions are:
# request_config
# update_config
string action

# Temoto namespace where the configuration belongs to.
string temoto_namespace

# configuration in YAML format
string config
